content/projects/first-personal-website/_index.md

```diff
@@ -44,43 +38,8 @@ Please submit a link to your website. DO NOT submit a link to your github repo.
 
 To learn more about Github Pages visit this link https://guides.github.com/features/pages/
 
-### Requirements:
+## Javascript instructions
+- http://syllabus.africacode.net/projects/first-personal-website/website-javascript/
```

Comment:
Try use content links to other content as opposed to static links, this has 2 benefits:
1 This makes formatting pretty on the frontend
2 this allows the linter to final and highlight any broken links for us to fix